pantheon-bundle/frontend/src/app/__snapshots__/searchBeta.test.tsx.snap

```diff
@@ -315,7 +320,7 @@ exports[`SearchBeta tests should render default Search component 1`] = `
                 className=""
```

Comment:
Probably not necessary, but I think it'd be nice?
I'd make this:
`repo-list filters-drawer__repo-list`

It's a new component, so I add the regular component class `repo-list`, then a class that lets us know it's parent component is the drawer `filters-drawer__repo-list`